source/utils/deprecation-warning.ts

```diff
@@ -0,0 +1,14 @@
+const alreadyWarned: string[] = [];
+
+export default (message: string) => {
+	if (alreadyWarned.includes(message)) {
+		return;
+	}
+
+	alreadyWarned.push(message);
+
+	// @ts-ignore Missing types.
+	process.emitWarning(message, {
```

Comment:
```suggestion
	process.emitWarning(`Got: ${message}`, {
```

maybe this?

Reply:
The output will be
`(node:29572) DeprecationWarning: Got: "options.rejectUnauthorized" is now deprecated, please use "options.https.rejectUnauthorized"`

Reply:
Yup. So people know it's from Got without enabling `--trace-warnings`.